Quaver.Shared/Screens/Edit/EditScreenView.cs

```diff
@@ -393,6 +393,10 @@ private void CreateGameplayPreview()
             
             // Makes it so that the playfield bookmark tooltips appear above preview.
             Playfield.Parent = Container;
+            
+            // Makes it so the selector goes above editor after enabling preview.
+            if(Selector != null)
```

Comment:
This null check shouldn't be required.

Reply:
It does, without it it still throws error. Unsure about other solution to it.

Reply:
What error does it throw? I've removed it and it works fine. The selector is created before the preview, so it should never be null.

Reply:
Cannot reproduce it anymore, but from what I remember it was "Object reference not set to an instance of an object"

Reply:
Well hopefully it wont be an issue, since when it was it prevented the editor to load at all and it was black screen.